dbtests/common_test.go

```diff
@@ -523,6 +540,48 @@ func testLoader(t *testing.T, db *sql.DB, dialect string, additionalOptions ...f
 			t.Errorf("cannot insert post: %v", err)
 		}
 	})
+
+	t.Run("SpannerConstraints", func(t *testing.T) {
```

Comment:
is it possible to move it to `TestSpanner`? Ideally it would be better than assertion in general test suite